third-party/py/pex/pex/common.py

```diff
@@ -269,17 +269,8 @@ def link(self, src, dst, label=None):
     self._ensure_parent(dst)
     abs_src = src
     abs_dst = os.path.join(self.chroot, dst)
-    try:
-      os.link(abs_src, abs_dst)
-    except OSError as e:
-      if e.errno == errno.EEXIST:
-        # File already exists, skip XXX -- ensure target and dest are same?
-        pass
-      elif e.errno == errno.EXDEV:
-        # Hard link across devices, fall back on copying
-        shutil.copyfile(abs_src, abs_dst)
-      else:
-        raise
+    safe_copy(abs_src, abs_dst, overwrite=False)
+    # If the file already exists, skip XXX -- ensure target and dest are same?
```

Comment:
We can actually avoid this path entirely if we just modify the code in `src/com/facebook/buck/python/make_pex.py` to prefer to copy on Windows.  I had originally wrote something like this, but @andrewjcg suggested changing Buck's code so it was fewer changes to PEX itself.